bin/lint_commit_msg.py

```diff
@@ -104,6 +105,19 @@ def lint_commit_msg(commit_msg_file, editor):
         print ''
         print '\n'.join(errors)
         print ''
+        if not editor:
+            print (
+                '\n'.join(textwrap.wrap(
+                    'You do not have EDITOR set in your environment. Please '
+                    'set EDITOR in your environment if you want to be able '
+                    'to edit the commit message. In the meantime, you can do:'
```

Comment:
I would say this sounds better - "Please set EDITOR in your environment if you want to be able to edit the commit message during the commit if it does not conform to the recommended format. Otherwise you can do ```git commit --amend``` to edit the commit message.